### PR TITLE
Maquetado final

### PR DESCRIPTION
**Pull request by manuelsalomon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <title>Maquetado</title>
+  </head>
+  <body>
+    <div class="content__">
```
Comment (santi698) [inline index.html]: Why do you suffix everything with `__`?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+html, body {
+  height: 100%;
+  width: 840px;
```
Comment (santi698) [inline style.css]: Use a wrapper div for setting width

File: style.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  height: 400px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header__{
+  width: 100%;
+  height: 6px;
+  content: "";
```
Comment (santi698) [inline style.css]: This has no effect

File: style.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 180px;
+}
+
+.triangle__{
+  width: 100%;
+  height: 12px;
+  content: "";
```
Comment (santi698) [inline style.css]: This has no effect

File: style.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  background-image: url("./assets/triangle.png");
+  background-repeat: repeat-z;
+}
+
+.footer__{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (santi698) [inline style.css]: This has too many rules, you may use `flex flex-align-center flex-justify-space-around` helper classes to extract some rules out of here

File: style.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  width: 300px;
+  height: 30px;
+  border-radius: 10px;
+}
+
+.btnSubs__{
+  padding: 10px;
+  color: white;
+  font-size: 16px;
+  background-color: Transparent;
```
Comment (santi698) [inline style.css]: All css default constants should be lowercase, `transparent` in this case

File: style.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.bar__{
+  margin-left: 40px;
+  margin-right: 40px;
+  width: 3px;
+  height: 334px;
+  content: "";
+  background: rgb(255,255,255); /* Old browsers */
+  background: -moz-linear-gradient(top, rgba(255,255,255,1) 0%, rgba(244,209,127,1) 30%, rgba(245,200,95,1) 50%, rgba(244,209,127,1) 70%, rgba(255,255,255,1) 100%); /* FF3.6-15 */
```
Comment (santi698) [inline style.css]: You can forget about browser prefixes for gradients in 2017 and just set `background: linear-gradient`

File: style.css [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+  background: -webkit-linear-gradient(top, rgba(255,255,255,1) 0%,rgba(244,209,127,1) 30%,rgba(245,200,95,1) 50%,rgba(244,209,127,1) 70%,rgba(255,255,255,1) 100%); /* Chrome10-25,Safari5.1-6 */
+  background: linear-gradient(to bottom, rgba(255,255,255,1) 0%,rgba(244,209,127,1) 30%,rgba(245,200,95,1) 50%,rgba(244,209,127,1) 70%,rgba(255,255,255,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#ffffff',GradientType=0 )
```
Comment (santi698) [inline style.css]: We don't need to support IE8 :smile:

File: index.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+      <div class="header__"></div>
```
Comment (santi698) [inline index.html]: use the more semantic `header` tag instead of div

File: index.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+      <div class="main__">
```
Comment (santi698) [inline index.html]: use the more semantic `section` tag for main and features

File: index.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+          <h4 class="title__"><strong>Generate reminders</strong></h4>
+          <p class="description__">Get reminded of who you owe money or who you owe money to.</p>
+        </div>
+        <div class="feature__">
+          <img src="./assets/ic_safe.png" alt="safe">
+          <h4 class="title__"><strong>No internet needed</strong></h4>
+          <p class="description__">All your dets are kept safe in your phone.</p>
+        </div>
+      </div>
+      <div class="footer__">
```
Comment (santi698) [inline index.html]: Use the `footer` tag

File: index.html (hunk 1 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 9 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 10 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 11 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 9 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 10 above)
Comment (manuelsalomon) [reply index.html]: solved

File: index.html (hunk 11 above)
Comment (manuelsalomon) [reply index.html]: solved

**Pull request closed without merging**